boards.txt

```diff
@@ -38600,6 +38600,198 @@ Geekble_ESP32C3.menu.EraseFlash.none.upload.erase_cmd=
+
+waveshare_esp32_s3_zero.menu.PartitionScheme.default=Default 4MB with spiffs (1.2MB APP/1.5MB SPIFFS)
+waveshare_esp32_s3_zero.menu.PartitionScheme.default.build.partitions=default
+waveshare_esp32_s3_zero.menu.PartitionScheme.defaultffat=Default 4MB with ffat (1.2MB APP/1.5MB FATFS)
+waveshare_esp32_s3_zero.menu.PartitionScheme.defaultffat.build.partitions=default_ffat
+waveshare_esp32_s3_zero.menu.PartitionScheme.no_ota=No OTA (2MB APP/2MB SPIFFS)
+waveshare_esp32_s3_zero.menu.PartitionScheme.no_ota.build.partitions=no_ota
+waveshare_esp32_s3_zero.menu.PartitionScheme.no_ota.upload.maximum_size=2097152
+waveshare_esp32_s3_zero.menu.PartitionScheme.noota_3g=No OTA (1MB APP/3MB SPIFFS)
+waveshare_esp32_s3_zero.menu.PartitionScheme.noota_3g.build.partitions=noota_3g
+waveshare_esp32_s3_zero.menu.PartitionScheme.noota_3g.upload.maximum_size=1048576
+waveshare_esp32_s3_zero.menu.PartitionScheme.noota_ffat=No OTA (2MB APP/2MB FATFS)
+waveshare_esp32_s3_zero.menu.PartitionScheme.noota_ffat.build.partitions=noota_ffat
+waveshare_esp32_s3_zero.menu.PartitionScheme.noota_ffat.upload.maximum_size=2097152
+waveshare_esp32_s3_zero.menu.PartitionScheme.noota_3gffat=No OTA (1MB APP/3MB FATFS)
+waveshare_esp32_s3_zero.menu.PartitionScheme.noota_3gffat.build.partitions=noota_3gffat
+waveshare_esp32_s3_zero.menu.PartitionScheme.noota_3gffat.upload.maximum_size=1048576
+waveshare_esp32_s3_zero.menu.PartitionScheme.huge_app=Huge APP (3MB No OTA/1MB SPIFFS)
+waveshare_esp32_s3_zero.menu.PartitionScheme.huge_app.build.partitions=huge_app
+waveshare_esp32_s3_zero.menu.PartitionScheme.huge_app.upload.maximum_size=3145728
+waveshare_esp32_s3_zero.menu.PartitionScheme.min_spiffs=Minimal SPIFFS (1.9MB APP with OTA/190KB SPIFFS)
+waveshare_esp32_s3_zero.menu.PartitionScheme.min_spiffs.build.partitions=min_spiffs
+waveshare_esp32_s3_zero.menu.PartitionScheme.min_spiffs.upload.maximum_size=1966080
+waveshare_esp32_s3_zero.menu.PartitionScheme.rainmaker=RainMaker 4MB
+waveshare_esp32_s3_zero.menu.PartitionScheme.rainmaker.build.partitions=rainmaker
+waveshare_esp32_s3_zero.menu.PartitionScheme.rainmaker.upload.maximum_size=1966080
+waveshare_esp32_s3_zero.menu.PartitionScheme.rainmaker_4MB=RainMaker 4MB No OTA
+waveshare_esp32_s3_zero.menu.PartitionScheme.rainmaker_4MB.build.partitions=rainmaker_4MB_no_ota
+waveshare_esp32_s3_zero.menu.PartitionScheme.rainmaker_4MB.upload.maximum_size=4038656
+
+waveshare_esp32_s3_zero.menu.PartitionScheme.otanofs=OTA no FS (2MB APP with OTA)
+waveshare_esp32_s3_zero.menu.PartitionScheme.otanofs.build.custom_partitions=partitions_otanofs_4MB
+waveshare_esp32_s3_zero.menu.PartitionScheme.otanofs.upload.maximum_size=2031616
+waveshare_esp32_s3_zero.menu.PartitionScheme.all_app=Max APP (4MB APP no OTA)
+waveshare_esp32_s3_zero.menu.PartitionScheme.all_app.build.custom_partitions=partitions_4MB
+waveshare_esp32_s3_zero.menu.PartitionScheme.all_app.upload.maximum_size=4128768
```

Comment:
This partitions says no OTA, but in the partitions there is ota_0 and ota_1.

```
# Name,    Type,  SubType,     Offset,      Size,       Flags
 nvs,       data,  nvs,         0x9000,      0x5000,
 otadata,   data,  ota,         0xe000,      0x2000,
 ota_0,     app,   ota_0,       0x10000,     0x1F0000,
 ota_1,     app,   ota_1,       0x200000,    0x1F0000,
```

So you can delete ota_1 and change the size of ota_0 to match flash size.

Reply:
ok,ill do it